t.css

```diff
@@ -0,0 +1,5 @@
+a {}
+@media {}
+
+b {}
```

Comment:
Expected empty line before comment.

```suggestion

/* comment */
```